docs/scenarios_authoring/BasicNodes.md

```diff
@@ -95,16 +95,22 @@ Split node doesn't have additional parameters.
 
 ![for_each](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_foreach0.png)
 
-`for-each` transforms the stream so that subsequent nodes are executed once for every value (possibly multiple times).
```

Comment:
Reworded to reduce the probability of reader thinking about for-each as the loop.